### PR TITLE
Rename objects and fields

### PR DESCRIPTION
**Pull request by AlvaroHuancaPerez**
The fields of objects were modified to the English language.

### CODE REVIEW

File: src/api-interfaces/lib/client.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
+export enum TypeDocumentEnum {
+    DNI = "DNI",
+    PASSPORT = "Pasaporte",
+    CE = "Carnet de Extranjeria",
+    RUC = "RUC"
+}
+
+export enum PositionEnum {
+    BULKING_AGENT = "Agente de Carga",
```
Comment (jpenam23) [inline src/api-interfaces/lib/client.ts:9]: we use CUSTOM_AGENT instead of BULKING_AGENT

File: src/api-interfaces/lib/client.ts [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    CUSTOMS_BROKER = "Agente de Aduana",
```
Comment (jpenam23) [inline src/api-interfaces/lib/client.ts:10]: CUSTOMS_ADUANA

File: src/api-interfaces/lib/client.ts [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    IMPORTER = "Importador",
+    EXPORTER = "Exportador"
+}
+
+
+
+export enum InformationTypeEnum {
```
Comment (jpenam23) [inline src/api-interfaces/lib/client.ts:17]: Remove the suffix "Enum", just "InformationType" .. here and for all the Enums

File: src/api-interfaces/lib/client.ts [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    COMPLAINT = "Quejas",
+    CLAIM = "Reclamos"
+}
+
+export enum BusinessEnum {
+    LOAD = "Carga"
+}
+
+export enum ClaimAreaEnum {
+    IMPORTS = "Importaciones",
```
Comment (jpenam23) [inline src/api-interfaces/lib/client.ts:27]: IMPORT
EXPORT

**Review by Jael860 — CHANGES_REQUESTED** [review]
Have you run prettier? Also make sure there are no linter errors.

**Pull request merged**